ckan/requirements.txt

```diff
@@ -56,6 +56,7 @@ MarkupSafe==2.1.1
 messytables==0.15.2
 newrelic==8.5.0
 nose==1.3.7
+numpy==1.23.5
```

Comment:
I wonder why this got added...

Reply:
It might be a new dependency for shapely?